### ISSUE TITLE
Last semicolon is missing

### ISSUE THREAD

**Issue by TrySound** [postcss/postcss-nested#9]
```scss
.embed-responsive {
	position: relative;
	display: block;
	height: 0;
	padding: 0;
	overflow: hidden;

	&:before {
		content: '';
		display: block;
		padding-bottom: 56.25%;
	}
}
```
No semicolon after `overflow: hidden`

**Comment by ai** [comment]
I don’t understand you :). What mean “no semicolon”? What do you do? What is CSS input? What is expected and real CSS output?

**Comment by ai** [comment]
I found a problem. Technically last child of `.embed-responsive` is `&:before` and it really has no last `;` :D.

For example

```css
.embed-responsive {
    position: relative;
    display: block;
    height: 0;
    padding: 0;
    overflow: hidden;

    &:before {
        content: '';
        display: block;
        padding-bottom: 56.25%;
    }

    margin: 0;
}
```

will get expected output.

**Comment by ai** [comment]
Fixed 75fb0e5936e9a6aea143f667b6163097b2e658c4

**Comment by TrySound** [comment]
Thanks! Will check tomorrow.

**Comment by ai** [comment]
Released in 0.3 version
